src/components/cylc/workflow/Toolbar.vue

```diff
@@ -119,7 +119,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       </v-btn>
       <v-btn icon size="small">
         <v-avatar color="primary" size="small">
-          {{ userInitials }}
+          <v-icon v-if="!user.initials" :icon="$options.icons.mdiAccountEdit"></v-icon>
```

Comment:
I'm not sure how initials could end up undefined, so I don't think it's strictly necessary to handle this case, but there's no harm in it.